.github/workflows/report-on-vulnerabilities.yaml

```diff
@@ -16,15 +16,50 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       results: ${{ steps.parse-results.outputs.results }}
-    steps:
-    - name: Scan for vulnerabilities
+    steps:    
+    - name: Get Branches Name
+      id: get-branches
+      shell: bash
```

Comment:
Default shell is always bash so I'd suggest removing this.

Reply:
@chipzoller ok i will remove this in next commit. thanks for reviewing.😊